src/js/countriesCodes.js

```diff
@@ -0,0 +1,264 @@
+const countryCode = document.getElementById('js_countryCode')
```

Comment:
Trzeba by było albo dodać ten plik jako `<script>` do `index.html`, albo wykonywać ten kod tylko w `app.js`.

Reply:
a dlaczego nie może być to w bundlu?

Reply:
Jak najbardziej powinno być.